components/server/src/workspace/workspace-starter.ts

```diff
@@ -702,6 +719,18 @@ export class WorkspaceStarter {
                     referrerIde: workspace.context.referrerIde,
                 },
             });
+        } else if (WithEditorContext.is(workspace.context)) {
+            this.analytics.track({
+                userId: user.id,
+                event: "custom_editor_referrer",
```

Comment:
I guess we can reuse `ide_referrer` here and extend code above to support `useLatest` as well?